jobs/component_jobs.groovy

```diff
@@ -132,7 +132,8 @@ repos.each { Map repo ->
           DEIS_REGISTRY=quay.io/ make docker-build ${dockerPush}
         """.stripIndent().trim()
 
-        shell new File("${WORKSPACE}/bash/scripts/commit_description_parser.sh").text
+        // only attempt to parse commit description if -pr job
+        isMaster ?: shell new File("${WORKSPACE}/bash/scripts/commit_description_parser.sh").text
```

Comment:
I assume the absence of anything between the `?` and the `:` means a `noop` for the truthy path of this ternary expression?